README.md

```diff
@@ -54,11 +52,15 @@ if err != nil {
 }
 ```
 
-### Creating a map of tags associated with keys
+### Creating tags and propagating them
 
-tag.Map is a map of tags. Package tags provide a builder to create tag maps.
+Package tags provide a builder to create tag maps and put it
```

Comment:
Package tags provides a builder to create and put tag maps into the current context.

Reply:
The package is also named `tag` not `tags` but I think `tags` is a better name FWIW

Reply:
In Go, there is this culture of singularizing the packages because the package name is always a part of the symbol name.

Then tag.Map reads like tag map.

Done.